packages/runtime/src/useCases/consumption/attributes/DeleteRepositoryAttribute.ts

```diff
@@ -39,7 +39,7 @@ export class DeleteRepositoryAttributeUseCase extends UseCase<DeleteRepositoryAt
         const ownSharedIdentityAttributePredecessors = await this.attributeController.getSharedPredecessorsOfRepositoryAttribute(repositoryAttribute);
         for (const ownSharedAttribute of [...ownSharedIdentityAttributes, ...ownSharedIdentityAttributePredecessors]) {
             if (!ownSharedAttribute.isOwnSharedAttribute(this.accountController.identity.address)) {
-                return Result.fail(RuntimeErrors.attributes.isNotOwnSharedAttribute(ownSharedAttribute.id));
+                throw RuntimeErrors.attributes.isNotOwnSharedAttribute(ownSharedAttribute.id);
```

Comment:
RuntimeErrors are for **outward facing** communication, so if we want the user to handle it. Please don't use this error, as the user then would think it's his fault.

Reply:
I think this comment got lost (is marked as outdated), but not resolved. Same opinion for the same issue.

Another question is: what do we do with the other attributes that were skipped in the process?

Reply:
Another problem could be that this might leave the attributes in an inconsisten state some with some without source attribute. Why dont we move all the Validation stuff in the validator ? so that we know we can proceed without error

Reply:
@jkoenig134 I adjusted the error. If you are not content with my changes, please be more specific about how to change it.

Reply:
My Idea was not to throw a runtime error (with ErrorCode) and instead just an Error

Reply:
I went with throwing an error without code now. Putting it in the validator would duplicate much logic, which doesn't seem clean to me. Also, if we care about this here now, I guess there are a lot of places existing code needs to be checked for throwing Errors (without code) vs returning failing results.